docs/device_report.md

```diff
@@ -1,14 +1,19 @@
-# Device 9a02571e8f01, 2019-06-29 00:05:28+00:00 to 2019-06-29 00:11:06+00:00
+# Device 9a:02:57:1e:8f:01, *** Make *** *** Model ***
+
+## Test Roles
 
 |  Role  |      Name              | Status |
 |--------|------------------------|--------|
 |Operator| *** Operator Name *** |        |
 |Approver| *** Approver Name *** |        |
 
-| Test iteration   |                        |
+## Test Iteration
+
+| Test             |                        |
```

Comment:
Column header doesn't seem quite right -- these aren't tests, it's information about the report. You might just be able to leave out the header completely?  I think it's really just "property" and "value" which isn't very interesting.

Reply:
Thanks for having a look! 
Yes, it could be something generic like 
| Item | Value |
as you suggest or revert to what you had put 
| Test iteration | |
I can leave the headers empty. If I remove the header, the table doesn't render unfortunately.
Happy to go with what you prefer, but I'll also discuss with Clare.
BTW, Travis is now happy! :-)